### DISCUSSION TITLE
Problem when deploying and updating a project in Vercel

### DISCUSSION THREAD

**Discussion by oscarjpicazo** [vercel/community#5729] [Help]
### Summary

Hello,
I am experiencing a problem with a project that I uploaded to Vercel, which is not being updated every time I commit/push locally, and which, if I try to upload again, also returns an error.

The error that appears in the Vercel logs when it returns an error when trying to deploy is:
```
node_modules/@types/react-redux/index.d.ts(6,5): error TS2305: Module '"react"' has no exported member 'JSX'.
Error: Command "npm run build" exited with 2
```

### Steps to Reproduce

The current deployment link:
https://feynman-tiktok.vercel.app/courses
(It works, but does not update the changes locally)

Importing JSX into "index.d.ts":
```
import {
    ClassAttributes,
    Component,
    ComponentClass,
    Context,
    JSX,
    JSXElementConstructor,
    NamedExoticComponent,
    ReactNode,
} from "react";

import { Action, AnyAction, Dispatch, Store } from "redux";

import hoistNonReactStatics = require("hoist-non-react-statics");
```

**Comment by mazenemam19** [comment]
I came across this today, did you find a solution?

**Reply by pawlean** [reply]
@mazenadel19 Happy to help! Do you get the same error?

**Reply by mazenemam19** [reply]
Hi @pawlean,

Thank you for your response. Yes, it's the same error I encountered. I managed to resolve it yesterday by updating my tsconfig.json file with the following settings:
```
{
  "compilerOptions": {
  "skipLibCheck": true,
   "esModuleInterop": true,
   "jsx": "react-jsx"
   }
}
```

**Reply by pawlean** [reply]
Thanks for sharing your solution @mazenadel19! Appreciate it on behalf of the community. Let us know if you have any other questions!

[a comment by an automated account is not shown]
